www/index.html

```diff
@@ -1,24 +1,24 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
-
-<head>
-    <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
     <title>{[app_name]s}</title>
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
-</head>
+  </head>
 
-<body>
+  <body>
     <script type="module">
-        {[imports]s}
+      {[imports]s}
 
-        import * as loader from "./loader.js";
-        const instance = await loader.load("{[wasm_path]s}.wasm", imports);
-        let wasm = instance.exports;
+      const response = await fetch("./{[wasm_path]s}.wasm");
+      const buffer = await response.arrayBuffer();
+      const module = await WebAssembly.instantiate(buffer, imports);
 
-        {[inits]s}
+      const wasm = module.instance.exports;
 
-        loader.init(instance);
-    </script>
-</body>
+      {[inits]s}
 
-</html>
\ No newline at end of file
+      wasm.main();
```

Comment:
I think the symbol for entry point should be something else....some OSes use ``_start`` I believe?